src/alchemlyb/parsing/amber.py

```diff
@@ -190,10 +192,10 @@ def file_validation(outfile):
             invalid = True
 
         mbar_ndata = 0
-        have_mbar, mbar_ndata = secp.extract_section('^FEP MBAR options:',
+        have_mbar, mbar_ndata, mbar_states = secp.extract_section('^FEP MBAR options:',
                                                       '^$',
                                                       ['ifmbar',
-                                                        'bar_intervall'],
+                                                        'bar_intervall', "mbar_states"],
```

Comment:
What is this change doing? Is it changing the functionality? If so, it needs an CHANGELOG and versionchanged entry.

Also, just for optical consistency use the same quote characters and add it on a new line.

Reply:
Does parsing this section affect reading performance?

Reply:
I don't think this will affect reading performance, but I hadn't tested it. We were already parsing the same line, but instead of reading 2 items now we read 3, not a big change. This is needed to check if there is consistency in the number of MBAR states read and expected by AMBER, so it's a value never exposed to the user and it doesn't change the overall functionality!

Reply:
ok, thanks for explaining